tests/functional/aws-node-sdk/test/bucket/putBucketNotification.js

```diff
@@ -0,0 +1,111 @@
+const assert = require('assert');
+const { S3 } = require('aws-sdk');
+
+const checkError = require('../../lib/utility/checkError');
+const getConfig = require('../support/config');
+const BucketUtility = require('../../lib/utility/bucket-util');
+
+const bucket = 'mock-notification-bucket';
+
+function getNotificationParams(events, arn, id, filter) {
+    const notifConfig = {
+        QueueConfigurations: [
+            {
+                Events: events || ['s3:ObjectCreated:*'],
+                QueueArn: arn || 'arn:scality:bucketnotif:::target1',
+            },
+        ],
+    };
+    if (id) {
+        notifConfig.QueueConfigurations[0].Id = id;
+    }
+    if (filter) {
+        notifConfig.QueueConfigurations[0].Filter = filter;
+    }
+    return {
+        Bucket: bucket,
+        NotificationConfiguration: notifConfig,
+    };
+}
+
+describe('aws-sdk test put notification configuration', () => {
+    let s3;
+    let otherAccountS3;
+
+    before(done => {
+        const config = getConfig('default', { signatureVersion: 'v4' });
+        s3 = new S3(config);
+        otherAccountS3 = new BucketUtility('lisa', {}).s3;
+        return done();
+    });
```

Comment:
Shouldn't need `done` if not using callbacks.
```suggestion
    before(() => {
        const config = getConfig('default', { signatureVersion: 'v4' });
        s3 = new S3(config);
        otherAccountS3 = new BucketUtility('lisa', {}).s3;
    });
```